### PR TITLE
[Release] Update versions for Release 11.0.0

### PR DESCRIPTION
**Pull request by ncooke3**
#no-changelog

### CODE REVIEW

File: FirebaseAnalyticsOnDeviceConversion.podspec [hunk 1]
```diff
@@ -18,7 +18,7 @@ Pod::Spec.new do |s|
 
     s.cocoapods_version = '>= 1.12.0'
 
-    s.dependency 'GoogleAppMeasurementOnDeviceConversion', '10.28.0'
+    s.dependency 'GoogleAppMeasurementOnDeviceConversion', '11.0'
```
Comment (ncooke3) [inline FirebaseAnalyticsOnDeviceConversion.podspec:21]: This needs to get fixed to preserve the patch.

File: ReleaseTooling/Sources/FirebaseReleaser/InitializeRelease.swift [hunk 2]
```diff
@@ -48,28 +46,39 @@ enum InitializeRelease {
       if pod.name == "Firebase" {
         updateFirebasePodspec(path: path, manifest: manifest)
       } else {
+        updatePodspecVersion(pod: pod, version: version, path: path)
+        
         // Pods depending on GoogleAppMeasurement and FirebaseFirestoreInternal specs
         // should pin the dependency to the new version.
-        for dep in ["GoogleAppMeasurement", "FirebaseFirestoreInternal"] {
-          updateDependenciesToLatest(dependency: dep, pod: pod, version: version, path: path)
+        if pod.name == "GoogleAppMeasurement" || pod.name == "FirebaseFirestoreInternal" {
```
Comment (ncooke3) [inline ReleaseTooling/Sources/FirebaseReleaser/InitializeRelease.swift:53]:
```suggestion
        if pod.name.hasPrefix("GoogleAppMeasurement") || pod.name == "FirebaseFirestoreInternal" {
```
```suggestion
        if pod.name == "GoogleAppMeasurement" || pod.name == "FirebaseFirestoreInternal" {
```

File: FirebaseAnalyticsOnDeviceConversion.podspec (hunk 1 above)
Comment (ncooke3) [reply FirebaseAnalyticsOnDeviceConversion.podspec:21]: Will be fixed by below suggestion.

Comment (ncooke3) [reply FirebaseAnalyticsOnDeviceConversion.podspec]: Pushed in 172e4fc. Need to tag and re-stage.

**Review by paulb777 — APPROVED** [review]
Please fix the style.

From a partial review, it looks like other tests may be addressed by other in-flight PRs

File: FirebaseAnalyticsOnDeviceConversion.podspec (hunk 1 above)
Comment (ncooke3) [reply FirebaseAnalyticsOnDeviceConversion.podspec]: Done.

**Comment by ncooke3** [comment]
> Please fix the style.
> 
> From a partial review, it looks like other tests may be addressed by other in-flight PRs

Thanks, @paulb777. I'm still picking through some errors I'd like to resolve before merging. The following podspecs won't stage:
- FirebaseDatabase.podspec
- FirebaseFirestoreInternal.podspec
- FirebaseFirestore.podspec
- Firebase.podspec

And I'm so far working through RTDB.

**Comment by ncooke3** [comment]
See update in https://github.com/firebase/firebase-ios-sdk/pull/13046#issuecomment-2145397219

Merging and will follow up with more CI fixes.

**Pull request merged**